src/pages/Vault/modals/StakeModal/index.spec.tsx

```diff
@@ -8,12 +8,12 @@ import MAX_UINT256 from 'constants/maxUint256';
 import fakeAccountAddress from '__mocks__/models/address';
 import { getBalanceOf, getAllowance } from 'clients/api';
 import renderComponent from 'testUtils/renderComponent';
-import useStakeWeiInVault from 'hooks/useStakeWeiInVault';
+import useStakeInVault from 'hooks/useStakeInVault';
 import en from 'translation/translations/en.json';
 import StakeModal, { IStakeModalProps } from '.';
 
 jest.mock('clients/api');
-jest.mock('hooks/useStakeWeiInVault');
+jest.mock('hooks/useStakeInVault');
```

Comment:
These don't work as I expect them. when declared like this. If we set them in in a `beforeAll` hook we can declare default return values in the mocked file. 

Then we can manually override the mock for specific tests.

Reply:
`jest.mock` calls get hoisted at the top of the tile, so putting these lines in a `beforeAll` block would produce the same result as right now